runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -5301,6 +5331,17 @@ static void jitStateLogic(J9JITConfig * jitConfig, TR::CompilationInfo * compInf
          }
       }
 
+#ifdef DEBUG_CODE_DISCLAIM
```

Comment:
I find it very useful since I don't need to run a separate process and there is no need to math two outputs - from java and from ps. Maybe we can keep it for now?